src/idea.js

```diff
@@ -5,13 +5,15 @@ class Idea {
     this.body = body;
     this.star = star || false;
   }
+
   saveToStorage() {
-    var stringifiedArray = JSON.stringify(ideaArray);
-    localStorage.setItem("storedIdeas", stringifiedArray);
+
   }
+
   deleteFromStorage() {
```

Comment:
I'm not sure how we can incorporate this. We tried refactoring but kept breaking other things.